TestPerChannelBookieClient.testEpollChannelTcpUserTimeout fails

**BUG REPORT**
After we committed https://github.com/apache/bookkeeper/commit/c8d4086697f85c9093f4da2c907a13e17c198914 I see this test failing very often on DataStax CI, that run the tests against current BookKeeper master, on JDK8.

***Describe the bug***
This is the error

[ERROR] Failures: 
[ERROR] org.apache.bookkeeper.proto.TestPerChannelBookieClient.testEpollChannelTcpUserTimeout
[ERROR]   Run 1: TestPerChannelBookieClient.testEpollChannelTcpUserTimeout:337 expected:<1236> but was:<1234>
[ERROR]   Run 2: TestPerChannelBookieClient.testEpollChannelTcpUserTimeout:337 expected:<1236> but was:<1234>
[ERROR]   Run 3: TestPerChannelBookieClient.testEpollChannelTcpUserTimeout:337 expected:<1236> but was:<1234>

This is the line
https://github.com/apache/bookkeeper/commit/c8d4086697f85c9093f4da2c907a13e17c198914#diff-271106bdb4f954bd92d6de6e38f2e9dfa542754e611001050350a72bd7a0b9acR338

--- a/bookkeeper-server/src/test/java/org/apache/bookkeeper/proto/TestPerChannelBookieClient.java
+++ b/bookkeeper-server/src/test/java/org/apache/bookkeeper/proto/TestPerChannelBookieClient.java
@@ -314,7 +314,7 @@ public class TestPerChannelBookieClient extends BookKeeperClusterTestCase {
         EventLoopGroup eventLoopGroup = new EpollEventLoopGroup();
         OrderedExecutor executor = getOrderedSafeExecutor();
         ClientConfiguration conf = new ClientConfiguration();
-        int tcpUserTimeout = 1234;
+        int tcpUserTimeout = 1236; // this value may be rounded on some Linux implementations
         BookieId addr = getBookie(0);
 
         // Pass to the PerChannelBookieClient object the client configuration with TCP user timeout.

